frontend/public/components/utils/units.js

```diff
@@ -223,7 +223,7 @@ export const humanizeSeconds = (v, initialUnit, preferredUnit) =>
   humanize(v, 'seconds', true, initialUnit, preferredUnit);
 export const humanizeCpuCores = (v) => {
   const value = v < 1 ? round(v * 1000) : v;
-  const unit = v < 1 ? 'm' : '';
+  const unit = v < 1 ? 'm' : 'millicores';
```

Comment:
This looks incorrect. If it's greater than 1, it is cores not millicores.

Reply:
We should be careful changing this since it affects *everywhere* we use this function. If we spell this out fully, let's evaluate how this looks in different parts of the UI. Note that `m` is the standard suffix and what's used in YAML, so there is a disadvantage to spelling it out as well.

Reply:
`humanizeCpuCores ` is used in too many places, since the function is only passed by reference, maybe create a separate one to use when `millicores` are useful?

Reply:
Also, `millicores` is longer than the other units we use, so it may not fit well on graph axes.

Perhaps it does make sense for this to be a separate function in addition to the existing `humanizeCpuCores`. Then calling code can explicitly indicate which format it requires.